src/core/config/mcp-registry.ts

```diff
@@ -0,0 +1,323 @@
+        requiresSetup: true,
+        tags: ['search', 'web', 'api', 'information'],
+    },
+
+    google_drive: {
+        id: 'google_drive',
+        name: 'Google Drive',
+        description: 'Access and manage Google Drive files and folders',
+        category: 'Cloud Storage',
+        config: {
+            type: 'stdio',
+            command: 'npx',
+            args: ['-y', '@modelcontextprotocol/server-gdrive'],
+            connectionMode: 'lenient',
+        },
+        setupInstructions: 'Requires Google Drive API credentials. Follow OAuth setup process.',
+        requiresSetup: true,
+        tags: ['google', 'drive', 'cloud', 'storage', 'files'],
+    },
+
+    slack: {
+        id: 'slack',
+        name: 'Slack',
+        description: 'Send messages and interact with Slack workspaces',
+        category: 'Communication',
+        config: {
+            type: 'stdio',
+            command: 'npx',
+            args: ['-y', '@modelcontextprotocol/server-slack'],
+            env: {
+                SLACK_BOT_TOKEN: '$SLACK_BOT_TOKEN',
+            },
+            connectionMode: 'lenient',
+        },
+        setupInstructions:
+            'Requires Slack Bot Token. Create a Slack app at: https://api.slack.com/apps',
+        requiredEnvVars: ['SLACK_BOT_TOKEN'],
+        requiresSetup: true,
+        tags: ['slack', 'communication', 'messaging', 'team'],
+    },
+
+    notion: {
+        id: 'notion',
+        name: 'Notion',
+        description: 'Manage Notion pages, databases, and content',
+        category: 'Productivity',
+        config: {
+            type: 'stdio',
+            command: 'npx',
+            args: ['-y', '@makenotion/notion-mcp-server'],
+            env: {
+                NOTION_API_KEY: '$NOTION_API_KEY',
+            },
+            connectionMode: 'lenient',
+        },
+        setupInstructions:
+            'Requires Notion Integration Token. Create one at: https://www.notion.so/my-integrations',
+        requiredEnvVars: ['NOTION_API_KEY'],
+        requiresSetup: true,
+        tags: ['notion', 'productivity', 'notes', 'database'],
+    },
+
+    docker: {
+        id: 'docker',
+        name: 'Docker',
+        description: 'Manage Docker containers, images, and compose services',
+        category: 'DevOps',
+        config: {
+            type: 'stdio',
+            command: 'npx',
+            args: ['-y', '@modelcontextprotocol/server-docker'],
+            connectionMode: 'lenient',
+        },
+        setupInstructions: 'Requires Docker to be installed and running on your system.',
+        requiresSetup: true,
+        tags: ['docker', 'containers', 'devops', 'deployment'],
+    },
+
+    kubernetes: {
+        id: 'kubernetes',
+        name: 'Kubernetes',
+        description: 'Manage Kubernetes clusters, pods, and resources',
+        category: 'DevOps',
+        config: {
+            type: 'stdio',
+            command: 'npx',
+            args: ['-y', '@kubernetes/mcp-server'],
```

Comment:
this server isn't real, let's make sure to test each of the servers we put in here (many of the others are also fake)